CHANGELOG.md

```diff
@@ -29,6 +29,12 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - Pass form id to donation form action url which help to show notices from session. (#5933)
 - Add hooks to legacy consumer to handle rendering, validating and saving for custom fields. (#5944)
 - Add min/max-length validation to text Fields API node types. (#5948)
+- Add min/max-length validation to text and textarea Fields API node types. (#5955)
```

Comment:
Since these are a part of the same task in the same release, we can merge them into a single item.

```suggestion
- Add min/max-length validation to text Fields API node types. (#5948, #5955)
```